### ISSUE TITLE
RuntimeError: Could not find libbitcoinkernel

### ISSUE THREAD

**Issue by ismaelsadeeq** [stickies-v/py-bitcoinkernel#2]
Thanks for this, @stickies-v.

After following the guide for the auto-installation of `py-bitcoinkernel`:
https://github.com/stickies-v/py-bitcoinkernel/tree/main?tab=readme-ov-file#installation

I tried importing `pbk`, but I am getting the following error:

```terminal
>>> import pbk
Traceback (most recent call last):
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/capi/bindings.py", line 43, in <module>
    _libraries['libbitcoinkernel'] = ctypes.CDLL(_find_bitcoinkernel_lib())
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/capi/bindings.py", line 33, in _find_bitcoinkernel_lib
    raise RuntimeError(
RuntimeError: Could not find libbitcoinkernel. Please either:
1. Install it in your system library path
2. Place it in ../lib/ relative to the Python package
3. Set the BITCOINKERNEL_LIB environment variable to the full path

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/__init__.py", line 1, in <module>
    from pbk.block import Block, BlockHash, BlockIndex, BlockUndo
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/block.py", line 4, in <module>
    import pbk.capi.bindings as k
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/capi/__init__.py", line 1, in <module>
    from pbk.capi.base import KernelOpaquePtr, KernelPtr
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/capi/base.py", line 3, in <module>
    import pbk.capi.bindings
  File "/home/abubakar-dev/py-bitcoinkernel/.venv/lib/python3.10/site-packages/pbk/capi/bindings.py", line 45, in <module>
    raise RuntimeError(f"Failed to load libbitcoinkernel: {e}")
RuntimeError: Failed to load libbitcoinkernel: Could not find libbitcoinkernel. Please either:
1. Install it in your system library path
2. Place it in ../lib/ relative to the Python package
3. Set the BITCOINKERNEL_LIB environment variable to the full path
```

Python version: 3.10  
OS: Ubuntu 22

**Comment by stickies-v** [comment]
Thanks a lot for trying it out and reporting this issue, this should now be fixed in #1 (specifically [this line](https://github.com/stickies-v/py-bitcoinkernel/pull/1/commits/0ebe71a0fa3c0148e1002bfe1c82767cd31ed499#diff-dc9a24a1e15a3a7cb66513898ff899ba95c9e45f3851d24dcb8b2fe37eda8524R22)). I'll be merging the fix shortly.

**Comment by ismaelsadeeq** [comment]
Yes I can confirm that this issue is fixed now after https://github.com/stickies-v/py-bitcoinkernel/pull/1
